src/test/java/org/apache/accumulo/access/AccessExpressionBenchmark.java

```diff
@@ -117,8 +118,8 @@ List<String> getStringExpressions() {
       return allTestExpressionsStr;
     }
 
-    public ArrayList<EvaluatorTests> getEvaluatorTests() {
-      return evaluatorTests;
+    public List<EvaluatorTests> getEvaluatorTests() {
+      return Collections.unmodifiableList(evaluatorTests);
     }
```

Comment:
I experimented with this locally. I was curious why it complained about this method that returned a list and not others that returned a list.  It was because the method was public, but it did not need to be public.  So the following change avoids the spotbugs complaint w/o introducing any runtime overhead.

```suggestion
    List<EvaluatorTests> getEvaluatorTests() {
      return evaluatorTests;
    }
```

Reply:
Ah, okay. I didn't realize that it didn't need to be public. Thanks!